core/domain/exp_services.py

```diff
@@ -1828,10 +1828,15 @@ def _create_change_list_from_suggestion(suggestion):
                 object.
     """
 
-    return [{'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
-             'state_name': suggestion.state_name,
-             'property_name': exp_domain.STATE_PROPERTY_CONTENT,
-             'new_value': [suggestion.state_content]}]
+    return [{
+        'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+        'state_name': suggestion.state_name,
+        'property_name': exp_domain.STATE_PROPERTY_CONTENT,
+        'new_value': {
+            'html': suggestion.suggestion_html,
+            'audio_translations': []
```

Comment:
Ooh that's a great catch. Yes it would clear existing translations. I'm not sure how to handle this at this point though because the translations would become invalid anyway -- it ties into our system for how to update content in general and it's not clear how that's going to happen yet. Basically, at which point do the existing translations get invalidated?

I think maybe we leave it for now (since there are no translations at the moment) but we have to make absolutely sure we revisit this at the point we are updating content / invalidating existing audio translations? This PR doesn't do this yet.